pycryptobot.py

```diff
@@ -827,6 +827,9 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
 
                     # execute a live market buy
                     state.last_buy_size = float(account.getBalance(app.getQuoteCurrency()))
```

Comment:
A general remark, not relate to that PR.
The `last_buy_size` looks to me a bit misleading. It could be just `balance`. Maybe for a future improvement
if people do agree.